src/class-parsely.php

```diff
@@ -428,6 +456,28 @@ public function get_options() {
 		);
 	}
 
+	/**
+	 * Returns the value of a nested option.
+	 *
+	 * @since 3.16.0
+	 *
+	 * @param string          $option  The option to get.
+	 * @param Parsely_Options $options The options to get the value from.
+	 * @return mixed The value of the nested option.
+	 */
+	public static function get_nested_option_value( $option, $options ) {
+		$keys  = explode( '[', str_replace( ']', '', $option ) );
+		$value = $options;
+
+		foreach ( $keys as $key ) {
+			if ( isset( $value[ $key ] ) ) {
+				$value = $value[ $key ];
+			}
+		}
+
+		return $value;
+	}
```

Comment:
Is this function being used somewhere? I tried to look for it in the codebase, but it seems that is never used anywhere.

Reply:
Ah, thanks! That file was not expanded and I missed it :) I have reviewed that file, and looks good to me, so still approved! Feel free to merge it.